couchbase/json-buckets.yml

```diff
@@ -0,0 +1,968 @@
+
+      - name: incr.misses
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.incr_misses[0]
+        type: long_gauge
+        label: increment misses
+        description: Number of increment misses
+
+      - name: mem.actual.free
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.mem_actual_free[0]
+        type: long_gauge
+        label: free memory
+        unit: bytes
+        description: Free memory
+
+      - name: mem.actual.used
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.mem_actual_used[0]
+        type: long_gauge
+        label: used memory
+        unit: bytes
+        description: Used memory
+
+      - name: mem.free
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.mem_free[0]
+        type: long_gauge
+        label: free memory
+        unit: bytes
+        description: Free memory
+
+      - name: mem.total
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.mem_total[0]
+        type: long_gauge
+        label: available memory
+        unit: bytes
+        description: Total available memory
+
+      - name: mem.used
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.mem_used[0]
+        type: long_gauge
+        label: memory usage
+        unit: bytes
+        description: Engine's total memory usage (deprecated)
+
+      - name: mem.used.sys
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.mem_used_sys[0]
+        type: long_gauge
+        label: system memory usage
+        unit: bytes
+        description: System memory usage
+
+      - name: misses
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.misses[0]
+        type: long_gauge
+        label: misses
+        description: Total number of misses
+
+      - name: ops
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.ops[0]
+        type: long_gauge
+        label: operations
+        description: Total number of operations
+
+      - name: page.faults
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.page_faults[0]
+        type: long_gauge
+        label: page faults
+        description: Number of page faults
+
+      - name: replication.docs.rep.queue
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.replication_docs_rep_queue[0]
+        type: long_gauge
+        label: replication docs rep
+
+      - name: replication.meta.latency.aggr
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.replication_meta_latency_aggr[0]
+        type: long_gauge
+        label: replicate meta latency
+
+      - name: rest.requests
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.rest_requests[0]
+        type: long_gauge
+        label: rest requests
+        unit: request/second
```

Comment:
this is a rate metric. may not be useful for us. If we have total requests metric we can derive rate in our reports. same for other rate metrics also

Reply:
X is 1 second. I only get the first value of array.
What is the interval of our agent? i can recalculate the real count.

Reply:
> What is the interval of our agent?

10 seconds

Reply:
i don't see alternative for this requests.